### PR TITLE
CF-308 Document object mapping

### PR DESCRIPTION
**Pull request by roman-verchikov**
Adds documentation for the following object mapping mechanisms:

 - External networks mapping (`ext_net_map`);
 - Nova object mapping (`override_rules`).

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

File: docs/user/configuration/object_mapping.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+--------------------
+
+Often clouds don't have the same availability zones or server groups defined,
+in these cases migration operator may want to map those objects in source 
+cloud to objects in destination cloud. This can be done by specifying 
+``[migrate] override_rules`` config option to point to a YAML file with the 
+following format::
+
+
+    <migration object>:
```
Comment (antonf) [inline docs/user/configuration/object_mapping.rst]: It's not <migration object> it's more like attribute of migration object

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**